projects/js-packages/licensing/components/activation-screen-controls/style.scss

```
@@ -24,117 +24,95 @@
 	@media screen and (min-width: 780px) {
 		padding: 64px;
 	}
-}
-
-.jp-license-activation-screen-controls--license-field,
-.jp-license-activation-screen-controls--license-field-with-error {
-	label {
-		font-weight: 600;
-		font-size: var(--font-body);
-	}
-
-	input.components-text-control__input {
-		border-radius: var(--jp-border-radius);
-		border: 1px solid #787c82;
-		font-size: 18px;
-		line-height: 24px;
-		margin: 0;
-		min-height: 48px;
-	}
-}
-
-.jp-license-activation-screen-controls--license-field {
-	input.components-text-control__input {
-		border: 1px solid #787c82;
-	}
-}
 
-.jp-license-activation-screen-controls--license-field-with-error {
-	input.components-text-control__input {
-		border: 1px solid var(--jp-red);
-	}
-}
 
-.jp-license-activation-screen-controls--license-field-error {
-	display: flex;
-	flex-direction: row;
-	align-items: start;
-	color: var(--jp-red);
-
-	svg {
-		margin-right: 4px;
-		fill: var(--jp-red);
-		min-width: 24px;
-	}
+	.jp-license-activation-screen-controls--license-field,
+	.jp-license-activation-screen-controls--license-field-with-error {
+		label {
+			font-weight: 600;
+			font-size: var(--font-body);
+		}
 
-	span {
-		font-size: var(--font-body);
+		input.components-text-control__input {
+			border-radius: var(--jp-border-radius);
+			border: 1px solid #787c82;
```

We could take this opportunity to update this to `var(--jp-gray-40)`.

```suggestion
			border: 1px solid var(--jp-gray-40);
```

---

somehow [that rule got duplicated](https://github.com/Automattic/jetpack/pull/21933/files#diff-da3770872c704019f9870616364978187e3490f5b9ea7a5098e49640d2697fe0R48) and I fixed the dupe 😆. On it